frontend/src/config/cambodia/layers.json

```diff
@@ -1448,9 +1448,9 @@
     "date_url": "",
     "feature_info_props": {
       "status": {
-        "type": "values_labels",
-        "label": "Sensor status",
-        "values": {
+        "type": "labelMapping",
```

Comment:
to be consistent let's try to keep things in snake_case in the json files and camelCase in the code.